<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAPTableExtractor</class>
 <widget class="QDialog" name="SAPTableExtractor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>623</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SAP Table Extractor</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>SAP Table</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SAPTable_lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Choose_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Fields</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>540</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Select Fields (Max. 512 Total Length)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Extract_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Extract Fields</string>
   </property>
  </widget>
  <widget class="QListWidget" name="Field_listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>371</width>
     <height>192</height>
    </rect>
   </property>
   <property name="selectionRectVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Restriction_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ABAP Restriction (eg. MATL_TYPE = 'HALB')</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Append_checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Append to existing table?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="maxRows_lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Rows</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DBName_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Database Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Database Table</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DBTable_lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dbselect_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select New Database</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="message_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>SAPTable_lineEdit</tabstop>
  <tabstop>Choose_pushButton</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
